### PR TITLE
chore(event_list): 12519 - show outdated event via link

### PR DESCRIPTION
**Pull request by IlyaIzr**
Old event not from the list are shown in separate slot in full state of events panel
image
mobile view
image

I know there are other ways to utilize this component, may be move resource out of it and create a component for single event that accepts the returned value of eventResourceAtom
The way i did it seems easy to read and consistent enough
But feel free to argue my approach

### CODE REVIEW

**Comment by propakov** [comment]
I found some new warnings, new component in in stack trace 
image

**Comment by propakov** [comment]
ah, it's src/features/events_list/components/EventCard/Analytics/Analytics.tsx line 71 
key should be unique it seems

File: src/features/events_list/components/EventFromResource/EventFromResource.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { useAtom } from '@reatom/react';
+import { ErrorMessage } from '~components/ErrorMessage/ErrorMessage';
+import { LoadingSpinner } from '~components/LoadingSpinner/LoadingSpinner';
+import { i18n } from '~core/localization';
+import { currentEventResourceAtom } from '~features/current_event/atoms/currentEventResource';
+import { createStateMap } from '~utils/atoms';
+import { EpisodeTimelineToggle } from '../EpisodeTimelineToggle/EpisodeTimelineToggle';
+import { EventCard } from '../EventCard/EventCard';
+
+export function EventFromResource({ hasTimeline }: { hasTimeline?: boolean }) {
```
Comment (propakov) [inline src/features/events_list/components/EventFromResource/EventFromResource.tsx]: as it's currentEvent maybe CurrentEventFromResource would be more concrete 🤔

**Review by propakov — APPROVED** [review]
Good job! 👍 
Maybe it's worth fixing these warnings in terms of this PR

File: src/features/events_list/components/EventFromResource/EventFromResource.tsx (hunk 1 above)
Comment (IlyaIzr) [reply src/features/events_list/components/EventFromResource/EventFromResource.tsx]: yeah, good suggestion!
JFYI concrete is a material 😄

**Comment by IlyaIzr** [comment]
> ah, it's src/features/events_list/components/EventCard/Analytics/Analytics.tsx line 71 key should be unique it seems

thanks for the catch! added key={nanoid(5)} instead

File: src/features/events_list/components/EventFromResource/EventFromResource.tsx (hunk 1 above)
Comment (propakov) [reply src/features/events_list/components/EventFromResource/EventFromResource.tsx]: yeah, true, but it's also could be used as "конкретный" thanslation =) 
asked native guy about this, maybe he tricked me :D

File: src/features/events_list/components/CurrentEventFromResource/CurrentEventFromResource.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { useAtom } from '@reatom/react';
+import { ErrorMessage } from '~components/ErrorMessage/ErrorMessage';
+import { LoadingSpinner } from '~components/LoadingSpinner/LoadingSpinner';
+import { i18n } from '~core/localization';
+import { currentEventResourceAtom } from '~features/current_event/atoms/currentEventResource';
+import { createStateMap } from '~utils/atoms';
+import { EpisodeTimelineToggle } from '../EpisodeTimelineToggle/EpisodeTimelineToggle';
+import { EventCard } from '../EventCard/EventCard';
+
+export function CurrentEventFromResource({ hasTimeline }: { hasTimeline?: boolean }) {
```
Comment (Akiyamka) [inline src/features/events_list/components/CurrentEventFromResource/CurrentEventFromResource.tsx]: Just CurrentEvent - postfix `resource` it for atoms that load something from api

File: src/features/events_list/components/ShortState/ShortState.tsx [hunk 3]
```diff
@@ -1,14 +1,12 @@
 import { Button, Text } from '@konturio/ui-kit';
 import { useAtom } from '@reatom/react';
 import { useEffect, useState } from 'react';
-import { ErrorMessage } from '~components/ErrorMessage/ErrorMessage';
-import { LoadingSpinner } from '~components/LoadingSpinner/LoadingSpinner';
 import { i18n } from '~core/localization';
 import { currentEventResourceAtom } from '~features/current_event/atoms/currentEventResource';
```
Comment (vkozio) [inline src/features/events_list/components/ShortState/ShortState.tsx]: Import from other feature, maybe you missed it

Comment (IlyaIzr) [reply src/features/events_list/components/ShortState/ShortState.tsx]: wow, i really did, thanks

**Comment by IlyaIzr** [comment]
Since we already have resource atoms in `/core/shared_state` folder and 3 components need to subscribe to `currentEventResourceAtom` I've moved it to `/shared_state`
I did not use `const [focusedGeometry] = useAtom(focusedGeometryAtom)` to show single event info because i also want to reflect error and loading states -> i need following resource atom

**Review by Akiyamka — APPROVED** [review]

**Comment by Akiyamka** [comment]
LGFM, but i have some glitch  in case use it with filter by bbox - sometimes we have selected event but in didn't show when list is empty

**Review by Akiyamka — APPROVED** [review]

**Pull request merged**